CONTRIBUTING.md

```diff
@@ -42,6 +42,7 @@ Thanks for contributing! In order to open a PR into the NB Defense project, you'
 
 1. Fork the repo
 2. Make your changes
+3. Optionally, follow the [pre-commit installation guide](https://pre-commit.com/#installation) to install NBDefense's pre-commit hooks. This will ensure your change passes our CI the first time.
```

Comment:
I actually just realized that `make install-dev` does this automatically. Maybe instead of this we can just hint to follow the "developing with NB Defense" instructions on your local copy of your fork?

Reply:
Agreed, done!